python/tskit/trees.py

```diff
@@ -1589,101 +1592,104 @@ def draw_text(
             )
         return str(text_tree)
 
+    # TODO when mimimum supported python version is 3.8, replace Optional[str] values
+    # with literals, e.g. SvgMaxTime = Union[Literal["tree", "ts"], int, None] = None
+
+    SvgMaxTime = Union[int, str, None]
+
     def draw_svg(
         self,
-        path=None,
+        path: Optional[str] = None,
         *,
-        size=None,
-        time_scale=None,
-        tree_height_scale=None,
-        max_time=None,
-        max_tree_height=None,
-        node_labels=None,
-        mutation_labels=None,
-        root_svg_attributes=None,
-        style=None,
-        order=None,
-        force_root_branch=None,
-        symbol_size=None,
-        x_axis=None,
-        x_label=None,
-        y_axis=None,
-        y_label=None,
-        y_ticks=None,
-        y_gridlines=None,
-        all_edge_mutations=None,
+        size: Optional[Tuple[int, int]] = None,
+        time_scale: "TreeSequence.SvgTimeScale" = None,
```

Comment:
I think we want to use ``from __future__ import typing`` to avoid some of this awkwardness with string types.

Reply:
Is that `from __future__ import annotations`? I think it's python 3.7, so do we want that as a minimum version?

Reply:
It's already in msprime